### ISSUE TITLE
combining flags doesn't seem possible

### ISSUE THREAD

**Issue by zoranpopovic** [podgorskiy/bimpy#7]
Combining flags doesn't seem possible since each flag is a class where or operator seems to fail     `bimpy.begin("Hi", flags=bimpy.WindowFlags.AlwaysAutoResize  | bimpy.WindowFlags.NoTitleBar)`

is there another way to do this?

**Comment by zoranpopovic** [comment]
my guess is that all combinable flags (not just WIndowFlags) need `py:arithmetic() parameter added in the enum definition, for example source should change to:
```
	py::enum_<ImGuiWindowFlags_>(m, "WindowFlags",  py:arithmetic())
```

**Comment by podgorskiy** [comment]
Thanks for reporting! Fixed in ab32019b52a564606c05a02e54a47c8e44eb3784
